SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The Drand team actively supports the most recent minor version of drand.
+Upcoming major or minor releases are announced with a prerelease at least one week in advance, and are communicated through the leagueofentropy@googlegroups.com email list.
+
+## Reporting a Vulnerability
+
+To report a security issue, please file an issue in this repository, and indicate that it is a vulnerability or security issue.
```

Comment:
I would simply say to directly send to the email . A  person shouldn't be the judge whether it's too serious of an issue or not to be disclosed publicly; by default it shouldn't be disclosed publicly, and it's up to us to judge whether that's a serious or not issue.